package.json

```diff
@@ -43,7 +46,7 @@
     "@govtechsg/decentralized-renderer-react-components": "^3.6.2",
     "@govtechsg/ethers-contract-hook": "^2.2.0",
     "@govtechsg/oa-encryption": "^1.3.3",
-    "@govtechsg/oa-verify": "^7.7.0",
+    "@govtechsg/oa-verify": "git+https://github.com/Open-Attestation/oa-verify.git#feat/add-networks",
     "@govtechsg/open-attestation": "^6.2.0",
```

Comment:
i see that oa-verify affected, which in turn dnsprove too. resolve those prs first?

Reply:
Okiee, will start with dnsprove first because oa-verify relies on it. Here's the PR for the changes in dnsprove: https://github.com/Open-Attestation/dnsprove/pull/25

Reply:
Updated in dnsprove, here's another PR to update the dnsprove version in oa-verify: https://github.com/Open-Attestation/oa-verify/pull/216